settings.gradle

```diff
@@ -12,6 +12,8 @@ include ':extensions:cloud:portability-cloud-local', ':extensions:cloud:portabil
 
 include ':extensions:auth:portability-auth-microsoft', ':extensions:auth:portability-auth-harness-microsoft'
 
+include ':extensions:auth:portability-auth-google'
```

Comment:
Lets also add this line to distributions/gateway-default/build.gradle

Reply:
Good to know.